lib/test/unit_test/DomainParticipant/DomainParticipant.cpp

```diff
@@ -0,0 +1,937 @@
+                    int32_t index) -> void
+            {
+                builtin::metatraffic_external_unicast_locators::update_cost(xml_file, profile_id, kind, index);
+            };
+    clear_functor_ = [](const std::string& xml_file, const std::string& profile_id, int32_t index) -> void
+            {
+                builtin::metatraffic_external_unicast_locators::clear_cost(xml_file, profile_id, index);
+            };
+
+    // Call test
+    print_push_update_clear_test();
+}
+
+TEST_F(DomainParticipantTests, builtin_metatraffic_external_unicast_locators_mask)
+{
+    // Test variables
+    // Default: UDP v4
+    valid_values_.push_back("1");
+    valid_values_.push_back("24");
+    invalid_values_.push_back("invalid");
+    invalid_values_.push_back("32");
+    invalid_values_.push_back("0");
+    invalid_messages_.push_back("does not match regular expression facet");
+    invalid_messages_.push_back("must be less than or equal to maxInclusive facet value");
+    invalid_messages_.push_back("must be greater than or equal to minInclusive facet value");
+
+    // Initialize functors
+    print_functor_ = [](const std::string& xml_file, const std::string& profile_id, int32_t index) -> std::string
+            {
+                return builtin::metatraffic_external_unicast_locators::print_mask(xml_file, profile_id, index);
+            };
+    push_functor_ = [](const std::string& xml_file, const std::string& profile_id, const std::string& kind) -> void
+            {
+                builtin::metatraffic_external_unicast_locators::push_mask(xml_file, profile_id, kind);
+            };
+    update_functor_ = [](const std::string& xml_file, const std::string& profile_id, const std::string& kind,
+                    int32_t index) -> void
+            {
+                builtin::metatraffic_external_unicast_locators::update_mask(xml_file, profile_id, kind, index);
+            };
+    clear_functor_ = [](const std::string& xml_file, const std::string& profile_id, int32_t index) -> void
+            {
+                builtin::metatraffic_external_unicast_locators::clear_mask(xml_file, profile_id, index);
+            };
+
+    // Call test
+    print_push_update_clear_test();
+}
+
+/**********************************************************************************************************************/
+/* DOMAIN PARTICIPANT TESTS                                                                                           */
+/* Builtin::InitialPeers                                                                                              */
+/**********************************************************************************************************************/
+TEST_F(DomainParticipantTests, builtin_initial_peers_kind)
+{
+    // Test variables
+    valid_values_.push_back("udp_v4");
+    valid_values_.push_back("udp_v6");
+    invalid_values_.push_back("tcp_v4");
```

Comment:
I think TCP is allowed in initial peers

Reply:
You are correct. Copy/pasting has the small detail of overlooking these details. Thanks!